scout/server/blueprints/variants/controllers.py

```diff
@@ -284,17 +293,17 @@ def render_variants_page(
         case_obj["_id"], form.data, None, category, build=genome_build
     )
 
-    if request.form.get("export"):
-        return download_variants(store, case_obj, variants_query)
-
-    data = sv_mei_variants(
+    data = decorator(
         store=store,
-        institute_obj=institute_obj,
-        case_obj=case_obj,
+        institute=institute_obj,
+        case=case_obj,
         variants_query=variants_query,
         page=page,
     )
 
+    if request.form.get("export"):
```

Comment:
Moved it down because the decorator above, when it's STRs, is responsible for sorting them as well. And we want to export them with the right sorting